src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -53,7 +53,7 @@ public static boolean validateEmail(String email, String location, Errors errs,
 
         if (!matcher.matches()) {
             setErrorMsgToLocation(errs, location, ValidationMessages.INVALID_EMAIL_ERROR_MESSAGE.replace("%s", location));
-            ApiLogger.infoContext(loggingContext, "Email address is not in the correct format for " + location);
+            ApiLogger.infoContext(loggingContext, "Enter an email address in the correct format for " + location);
```

Comment:
Not sure why log message content has changed here - I prefer the original content as the new one sounds more like a message that you would display to a user of a UI client.

Reply:
Changed it back.